src/printer.js

```diff
@@ -1420,15 +1420,30 @@ function printStatement(path, options, print) {
             ])
           )
         : "";
+
+      const hasEmptyBodyContents =
```

Comment:
Nitpick: I'd call it `hasEmptyBody` (same as below)

Reply:
@czosel :+1: